### DISCUSSION TITLE
Align existing model with one star + no tracking environment update

### DISCUSSION THREAD

**Discussion by nickym998** [mworion/MountWizzard4#277] [Q&A] [answered]
Hi,

I was wondering if there is an option to get MW4 to align an existing model doing a plate solve with one star?  Per's Model maker had an option to do this, it would just change the sync behaviour to align the whole model rather than adding a sync point.  Is this possible in MW4?

Also I use Per's stickstation which works fine but have a question about the "no tracking" environment update method.  If my imaging software does a dither at the end of an exposure, would this cause the pressure and temp to be updated using the "no tracking" option?  Dither is a mini slew so I think that the mount would not be tracking then?

many thanks

Nick

**Comment by Ruediger3** [comment, marked as answer]
> Hi,
> 
> I was wondering if there is an option to get MW4 to align an existing model doing a plate solve with one star? Per's Model maker had an option to do this, it would just change the sync behavior to align the whole model rather than adding a sync point. Is this possible in MW4?

> Nick

Hi Nick,

This function is already available, if I got your request properly. You may have a look here:
@mworion : is this alignment permanent, or does it get reset on the next mount boot? Is it modifying the stored model?

CS
Rüdiger

![2022-09-01_11h17_22](https://user-images.githubusercontent.com/65772654/187878893-e3a56379-dd5e-4024-bfab-93ce2e6be9b9.png)

**Reply by mworion** [reply]
Hi Nick, Rüdiger,

the mentioned function from MW4 does exactly this. It adjusts the existing model with a platesolve. From the description of the commend protocol this alters the somehow the model as correction will be shifted. So my guess is, that this will be persistent also after next boot. This only could improve the model fit in a very small area!

My recommendation would be - if this is used - to store the model before and reload it if this correction should be removed again. Still the point: if you needed the correction it might be necessary to redo an overall model anyway. 

Hope this helps.

Michel

**Comment by Ruediger3** [comment]
Hi Michel, 
I use this function quite often for minimal adjustments, which are related to thermal effects on the pillar. But I noticed, when applying it multiple times, the model gets degraded step by step. This would support your assumption that it is persistent. But I think this is worth a question to 10M support.

**Comment by nickym998** [comment]
Thanks for the replies, much appreciated.

Any insight on the pressure and temp update question?

Kind regards,

Nick

**Comment by mworion** [comment]
Oh I missed the point about dither. As you said it depends how the dither is done. If the imaging application does it with mini slews -> update should work. If you use the dithering of the 10micron itself -> it might not. 
I would recommend to switch to continuous ans the mount filters the values in that case anyway. The very point about it is, that you do not get an irregular step during an integration.
